### PR TITLE
Feature/resume upload be

### PR DESCRIPTION
**Pull request by jts7382**
can upload resumes to the server. There is no way to view it in the app. can find in uploads folder

### CODE REVIEW

File: backend/build.gradle.kts [hunk 1]
```diff
@@ -59,6 +59,10 @@ tasks.register("setupEnvironmentVariables") {
         if (System.getenv("CRD_DB_PASSWORD") == null) {
             throw IllegalStateException("Required environment variable is not defined: CRD_DB_PASSWORD")
         }
+
+        if (System.getenv("FILE_UPLOAD_DIRECTORY") == null) {
```
Comment (bml2238) [inline backend/build.gradle.kts:63]: is this a variable we need to do in setup? If so, can you update the readme accordingly?

Comment (jts7382) [reply backend/build.gradle.kts]: I just added a default

**Review by bml2238 — APPROVED** [review]
just a note but make sure the api docs are up to date with the implementation. it doesn't necessarily need to be functional from swagger but as long as the documentation matches up with the requirements for the actual function

**Review by rcb2957 — APPROVED** [review]
Works

**Pull request merged**